app/views/users/show/_guides.html.erb

```diff
@@ -20,10 +20,10 @@
               id: guide.crop.id }, :class => 'action show crop-name') %>
           </td>
           <td>
-            <%= link_to(icon('edit'), { :controller => 'guides', :action => 'edit', :id => guide.id}, :class => 'action show') %>
+            <%= link_to('<i class="fa fa-edit"></i> Edit'.html_safe, { :controller => 'guides', :action => 'edit', :id => guide.id}, :class => 'action show') %>
           </td>
           <td>
-            <%= link_to(icon('trash'), guide_path(guide), :method => 'delete', :class => 'action show', data: {confirm: "Are you sure?"}) %>
+            <%= link_to('<i class="fa fa-trash"></i> Delete'.html_safe, guide_path(guide), :method => 'delete', :class => 'action show', data: {confirm: "Are you sure?"}) %>
```

Comment:
This is a great start, there's just some domain things you should know about! 

We use the [font-awesome-sass](https://github.com/FortAwesome/font-awesome-sass) gem and Rails' translate.

So you can do something like this (untested):

```
<%= link_to(icon('edit') + ' ' + t('application.edit'), guide_path(guide), title: t('application.edit'), method: 'delete', class: 'action show') %>
```

You'll probably need to add the translation text to the appropriate place in `en.yml`.